blog/district-c-collaboration/index.md

```diff
@@ -0,0 +1,201 @@
+
+- Complex products and systems
+- Volume of information
+- Gathering insights from experts and seeking feedback
+- Building knowledge about the Hitachi Energy products
+
+![](images/soap_box.jpg) **Lance, Quinn, and Andrzej tell a story about the
+project at the soap! content conference in June, 2023.**
+
+## How it was done
+
+First, there were two rounds of Zoom meetings and presenting information.
+
+The members of the Hitachi Energy Team explained what it is like to be a
+Technical Writer and why this role requires specialized expertise. Concise and
+clear data gave insight into the organization's daily work and critical focus
+areas.
+
+Then, students pondered what was said and came back with additional questions to
+dig deeper. Each student group had a specific amount of time to ask strategic
+questions. These were intense, focused sessions in which both sides tried to use
+the available time as best they could.
+
+At that point, this exercise was already beneficial to the Hitachi Energy Team.
+We all knew the specifics of our daily operations well. Still, it was a useful
+exercise to describe our role to someone outside of our working environment.
+Explaining without relying on jargon, project names, and mental shortcuts was
+challenging. It helped us to reflect on our needs and better understand them by
+clear formulation.
+
+## The big event
+
+Student groups learned what operating in a modern global corporate workplace is
+like. They gathered input, applied design thinking principles to empathize,
+identified opportunities for growth, and suggested sound solutions, which were
+then presented during the final **pitch meeting** held over Zoom.
+
+![](images/hitachi_crew.png)
+
+**Pitch meeting** is a presentation where an individual or a team presents a
+business idea or concept to potential investors, stakeholders, or clients to
+secure funding, support, or interest in the project. The presentation usually
+includes an overview of the product or service and the proposed solution.
+
+## Examples of solutions
+
+Each group presented its assessment of the opportunities and provided practical
+recommendations. Fresh perspectives and solutions were appreciated, as they
+logically tied to the identified areas for improvement. At the same time, the
+Hitachi Energy team offered practical feedback, rooting in business operations,
+experience, and practice.
+
+Students focused mainly on the nature of the interactions between the Technical
+Writers and the Subject Matter Experts (SME).
+
+They then explored many valid questions:
+
+- What is the most efficient way to get SME insights and build product
+  knowledge?
+- Are multiple team members often asked the same questions?
+
+One suggestion was to invite all SMEs to a single session to get all their
+answers. Alternatively, this could be resolved asynchronously through Teams or a
+survey.\
+\
+Personally, District C inspired me to use Microsoft Loop, allowing me to engage
```

Comment:
Co tutaj robi ten akapit? Czy to cytat? Brakuje tu jakiegoś wstępu albo wyjaśnienia.

I co oznaczają te dwa znaki `\` powyżej?

Reply:
ok, zastanowie sie czy usunac ten akapit albo umiescic go inaczej. akapit pokazuje przyklad decyzji podjetej na podstawie sugestii od studentow. Wiele osob komentuje ten akapit, wiec faktycznie jest do przemysenia na nowo.

Znaki \ dodal mi Pandoc ktorym konwertowalem z Worda. Usunalem je w calym tekscie.